packages/webapp/src/components/Locations.tsx

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import { observer } from 'mobx-react-lite'
+import * as React from 'react'
```

Comment:
You don't need to import React as a global anymore

Reply:
applied cleanup